.github/workflows/frontend.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+name: frontend
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - ui/**
+      - backport/**
+
+jobs:
+  workspace-tests:
+    runs-on: ubuntu-20.04
+    container:
+      image: docker.mirror.hashicorp.services/circleci/node:16-browsers
+      options: --user root
+    defaults:
+      run:
+        working-directory: ui
+    steps:
+    - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # pin@v3.3.0
+
+    # Install dependencies.
+    - name: install yarn packages
+      working-directory: ui
+      run: make deps
+
+    - run: make test-workspace
+
+  node-tests:
+    runs-on: ubuntu-20.04
+    container:
+      image: docker.mirror.hashicorp.services/circleci/node:16-browsers
+      options: --user root
+    steps:
+    - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # pin@v3.3.0
+
+    # Install dependencies.
+    - name: install yarn packages
+      working-directory: ui
+      run: make deps
+
+    - run: make test-node
+      working-directory: ui/packages/consul-ui
+
+  ember-build-test:
```

Comment:
Have we confirmed the following?
- we don't need to run OSS and ENT versions of these tests?
- just running the yarn test coverage out of [these existing CI tests](https://github.com/hashicorp/consul/blob/main/.circleci/config.yml#L800-L857) is ok?  IE, we are no long running the ember exam command.

If this is ok with the UI team, can you update the PR description to call this out, please?

Reply:
>we don't need to run OSS and ENT versions of these tests?

These jobs were duplicated and the only difference was the value of `CONSUL_NSPACES_ENABLED`, which can be controlled independently in the ENT repo

>IE, we are no long running the ember exam command

We are still running the exam command here: https://github.com/hashicorp/consul/pull/16731/files#diff-ffe3a297ed15e9a79846161a3731a580511bf13c6b5cc5a9e03ddcbd8515fdeeR69